dbt_platform_helper/commands/copilot.py

```diff
@@ -467,3 +468,57 @@ def _rmdir(path):
         for f in svc_addons_dir.iterdir():
             if f.is_file():
                 f.unlink()
+
+
+@copilot.command
+def vpc_generate():
```

Comment:
Is this being added to the general `platform-helper generate` command?

Reply:
I think it's related to the question of whether we generate env manifest files as part of this command. If we roll it into this `generate` command, then we need to either run `copilot env init` as part of `generate` or document that `copilot env init` will need to be run before `generate`

Reply:
Maybe we can move this command into an environment space...

`platform-helper environment update-vpc-config <environment>`

I agree that it should not be dong `copilot env init`. In fact, we will not need to run that to generate the environment manifests.

I would expect this command to be fired later from `platform-helper environment generate <environment>`.